server/app/serializers/fusor/deployment_serializer.rb

```diff
@@ -21,6 +21,7 @@ class DeploymentSerializer < ActiveModel::Serializer
                :openstack_undercloud_ip_addr,
                :openstack_undercloud_user,
                :openstack_undercloud_user_password,
+               :is_autogenerate_overcloud_password,
```

Comment:
This DB column might be a bit confusing.  All the others start with "openstack_".  Also "is" is usually part of getter/setter methods, not usually included in the db column.  It's kind of long, but how about using "openstack_overcloud_autogenerate_password" ?  We can still alias it client side on the template to "autogeneratePassword"